libbeat/dashboards/modify_json.go

```diff
@@ -42,24 +42,29 @@ type JSONFormat struct {
 }
 
 func ReplaceIndexInIndexPattern(index string, content common.MapStr) common.MapStr {
```

Comment:
Not directly related, I have just reorganized the code to have less nesting levels in these functions. Result should be the same.

Reply:
++ on the cleanup.